examples/Clustering_for_transaction_classification.ipynb

```diff
@@ -399,7 +398,7 @@
     "        .sample(transactions_per_cluster, random_state=42)\n",
     "        .values\n",
     "    )\n",
-    "    completion = client.chat.completions.create(\n",
+    "    response = openai.chat.completions.create(\n",
```

Comment:
revert to client cause you already define the client in 2nd cell

Reply:
Not sure what happened there, reverting this change.